intermediate-python-testing.md

```diff
@@ -295,7 +297,9 @@ def test_do_a_thing(mocker):
     # test things were done as expected
 ```
 
-If that's not tantalizing enough, you can also use `mock` to raise exceptions to test error handling, or simply turn off state-altering parts of your code that aren't relevant to the test at hand. Finally, `mock` keeps track of whether and how mocked methods are called, so you can test how your code is used (called _n_ times, or with this or that argument), without necessarily having to run it.
+Tantalizing, right? 
+
+You can also use `mock` to raise exceptions to test error handling, simply turn off state-altering parts of your code that aren't relevant to the test at hand, or return a response with JSON data (as we do with [MagicMock in Metro](https://github.com/datamade/la-metro-councilmatic/blob/master/tests/test_events.py#L117)). Finally, `mock` keeps track of whether and how mocked methods are called, so you can test how your code is used (called _n_ times, or with this or that argument), without necessarily having to run it.
```

Comment:
I think I'd phrase this like "You can also use mock to raise exceptions to test error handling, return canned website responses without hitting a live endpoint (as we do in Metro), or simply turn off state-altering parts of your code that aren't relevant to the test at hand."